operations/iuf/workflows/managed_rollout.md

```diff
@@ -59,15 +59,30 @@ referred to as the "canary node" in the remainder of this section. Use `--limit-
 
 1. Verify the canary node booted successfully with the desired image and CFS configuration.
 
-1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes, rebooting the nodes in the default staged manner in conjunction with the workload manager. If an
-immediate reboot of compute nodes is desired instead, add `-mrs reboot` to the `iuf run` command.
+1. Invoke `iuf run` with `-r` to execute the [`managed-nodes-rollout`](../stages/managed_nodes_rollout.md) stage on all nodes. This will stage data to BOS and allow the workload manager to reboot the nodes when it is ready to do so. The workload managers\
+must be configured to tell BOS to reboot nodes using this staged data. See ['Using staged sessions with Slurm'](<../../boot_orchestration/Rolling_Upgrades.md#staged-sessions-with-slurm>) for this configuration.  This assumes Slurm as the workload manager.\
+If an immediate reboot of compute nodes is desired instead, add `-mrs reboot` to the `iuf run` command.
 
     (`ncn-m001#`) Execute the `managed-nodes-rollout` stage.
 
     ```bash
     iuf -a "${ACTIVITY_NAME}" run -r managed-nodes-rollout
     ```
 
+1. Tell the workload manager to reboot the compute nodes. To reboot all compute nodes use the keyword ALL. Otherwise, compute nodes must be specified by NID.
+
+    (`ncn-m001#`) A sample reboot command to reboot all compute nodes:
```

Comment:
You'll definitely want to follow up on this with Slurm/WLM folks, but since you asked me about it in slack, I think you can either run this on UANs or maybe in the `slurmctld` pod. I don't know if the latter is a good idea or not, but it might be an option if we can't get to the UANs. E.g.:

```
ncn-m001:~ # kubectl exec -it -n user slurmctld-55d4668f6f-lxjtt -c slurmctld -- /bin/bash
slurm@slurm-host:/> scontrol --help
scontrol [<OPTION>] [<COMMAND>]
...
```

Reply:
I talked with David Gloe, and he said you can run scontrol either on the UANs or in the slurmctld pod.